config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  root 'root#index'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      resources :greetings, only: [:index]
+    end
+  end
+  # Defines the root path route ("/")
+  # root "articles#index"
+end
```

Comment:
- [ ] Please create a root route  (should be marked as `root`)

Reply:
Hi again @GzimAsani, I believe you missed **line 2**, there's a root route defined :+1:

Reply:
- Yep, I saw on line 10 as commented out and missed the one on second line :disappointed: